slyd/slyd/utils/storage.py

```diff
@@ -42,31 +122,87 @@ def move(self, old_file_name, new_file_name, allow_overwrite=False):
     def rmtree(self, name):
         shutil.rmtree(self.path(name))
 
+    def _save(self, name, content):
```

Comment:
This is being duplicated in `portia_server`. Can we import classes from that directory or is there some reason to keep these separate?